README.md

```diff
@@ -21,6 +21,17 @@ cargo-get uses rustfmt for formatting and clippy for linting.
 
 ### Installation
 
+### Pre-built Binaries
+
+Download the latest release from the following link:
+
+<https://github.com/nicolaiunrein/cargo-get/releases/latest>
+
+Place the binary in your `$PATH` and rename it to `cargo-get`.
```

Comment:
The syntax `cargo get something` only works if it is in the cargo bin directory as far as I am concerned.